README.md

```diff
@@ -52,6 +52,12 @@ Paste the following on your shell and you're good to go:
 $ curl -Lo- "https://raw.githubusercontent.com/bpkg/bpkg/master/setup.sh" | bash
 ```
 
+Or
+```sh
+$ curl -Lo- "https://coding.net/u/hydra1983/p/bpkg/git/raw/master/setup.sh" | \
```

Comment:
I don't think we should suggest installing bpkg from your fork.

Reply:
Sorry about this, I will remove it :D